layouts/shortcodes/twocol.html

```diff
@@ -0,0 +1,8 @@
+{{- $second_col_width := .Get "colsplit" | default "4" -}}
```

Comment:
is `$second_col_width` used in this file?

Reply:
Nope - good catch! That's left over from original figure short code. I'll remove it.